python/nano/test/pytorch/tests/test_bf16_ipex.py

```diff
@@ -35,30 +36,43 @@ def test_bf16_pytorch_less_1_10(self):
             trainer.quantize(model, precision='bf16', use_ipex=True)
 
 
-class Pytorch1_12:
-    def test_bf16_common(self):
-        """
-        Debug mode. Allow run bf16 forward without bf16 instruction support.
-        """
+class CaseWithoutAVX512:
+    @patch('bigdl.nano.deps.ipex.ipex_inference_bf16_model.PytorchIPEXJITBF16Model._check_cpu_isa', new_callable=PropertyMock)
```

Comment:
`CaseWithoutAVX512` will be run only when `check_avx512() == False` which means `_check_cpu_isa=False` by default. Is this mock necessary?